scripts/deploy.js

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const fs = require('fs');
+const child_process = require('child_process'); // eslint-disable-line camelcase
+const inquirer = require('inquirer');
+const pkg = require('../package.json');
+
+process.on('unhandledRejection', reason => console.log(reason));
+(async function() {
+  const currentVersion = pkg.version;
+
+  console.log(`\nCurrent version: ${currentVersion}\n`);
+
+  await askUnskippableQuestion(
+    `Have you verified TypeScript definitions file? [typescript/raven.d.ts]`
+  );
+  await askUnskippableQuestion(
+    `Have you updated changelog file with recent features and bugfixes? [CHANGELOG.md]`
+  );
+
+  const nextVersion = await inquirer
+    .prompt({
+      name: 'bump',
+      message: 'Which version part do you want to update?',
+      type: 'list',
+      choices: ['major', 'minor', 'patch'],
+      default: 'patch'
+    })
+    .then(
+      answers =>
+        new Promise((resolve, reject) => {
+          const [major, minor, patch] = currentVersion
+            .split('.')
+            .map(x => parseInt(x, 10));
+
+          switch (answers.bump) {
```

Comment:
In what cases would you ever want a major/minor bump that retained the latter bits?

Like, 1.9.3 > 2.9.3? When doing a major/minor, it should reset the latter bits to 0, so that going from 1.9.3 yields 2.0.0 or 1.10.0.

Reply:
You are 100% correct. Thanks! :)

Reply:
Updated